core-scroll-header-panel.html

```diff
@@ -337,7 +357,35 @@
       this.y = y;
       
       if (event) {
-        this.fire('scroll', {target: this.scroller}, this, false);
+        this.fire('scroll', {target: this.scrollObject}, this, false);
+      }
+    },
+
+    fullWindowSetup: function() {
+      this.scrollObject && this.scrollObject.removeEventListener('scroll', this._scrollHandler);
+      this.scrollObject = this.fullWindow ? window : this.$.mainContainer;
+      this.scrollObject.addEventListener('scroll', this._scrollHandler);
+      if (this.fullWindow) {
+        this.style.position = 'static';
+        this.style.height = 'auto';
+        if (!this.drawerPanel) {
```

Comment:
This ties this element closely to the core-drawer-panel element. Not sure if that's what we want.